packages/flame_3d/lib/src/components/group_3d.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flame_3d/components.dart';
+
+/// {@template group_3d}
+/// [Group3D] is a [Component3D] that acts as a container for other
+/// [Component3D]s.
+///
+/// Changing the position, rotating or scaling the [Component3D] will affect
+/// the whole group as if it was a single entity.
+///
```

Comment:
```suggestion
```

Reply:
they actually all seem to have this extra line. I can remove en mass if you don't want it

Reply:
Yes please!